src/picasso-definition/components/point-labels/index.js

```diff
@@ -34,6 +34,29 @@ export default function createPointLabels({ layoutService, themeService }) {
       fill: color,
       backgroundColor: style.backgroundColor,
     },
+    rendererSettings: {
+      transform: () => {
```

Comment:
I see the transform function is identical in the point and label component. I suggest we move that function into view-handler as well. In there we will have "direct" access to `isInteractionInProgress` after you move that in there, and the viewState! Will be nice and clean!